### ISSUE TITLE
Sidebar won't be colored

### ISSUE THREAD

**Issue by FuckingToasters** [CyanVoxel/Obsidian-Colored-Sidebar#12]
Hi there!

I've followed the ReadMe Tutorial and installed the custom css snippet.
I then added those Prefixes in front of my Notes which sadly didn't change anything.
![grafik](https://github.com/user-attachments/assets/683bb4f7-7c60-403e-b2cf-8872e374d894)

**Comment by BlinkTwice12** [comment]
Im having the same issue any luck figuring out ?

**Comment by Rex9988** [comment]
> Im having the same issue any luck figuring out ?

I'm not sure if both of you have figured it out yet, but just in case, if I'm looking at that image correctly, you're trying to color a note, these colors will only affect folders however.
Create a new folder and name it with the correct prefix to get the color you want.
At least as far as I know, that's how I think it works.

**Comment by thorrrr** [comment]
My folders are not being coloured  

![image](https://github.com/user-attachments/assets/436bbd83-ec67-4774-992f-7ceb5e3e613a)
![image](https://github.com/user-attachments/assets/65c081df-68b3-41cb-b4c5-e93dbf6a7d3b)
![image](https://github.com/user-attachments/assets/6110f4af-62f9-423b-95cd-a7264596e394)

No matter what i do can i get it to work !

**Comment by Rex9988** [comment]
Ah I think I see what your problem is.
What you did is inside your vault you created a folder called 'Vault' and then inside THAT folder is where you're trying to color the folders, that won't work, I just tested it.

The color needs to be applied to the TOP most folder, only then will it work.
Try making a new folder outside of that vault folder or renaming the vault folder itself to one of the colors (I repeat, the vault folder, NOT the vault itself.)
Then it'll be whichever color it was selected for it and all subsequent folders inside it.
As far as I know you can't color sub folders, only top ones, at least with this snippet.

**Comment by FuckingToasters** [comment]
> Ah I think I see what your problem is. What you did is inside your vault you created a folder called 'Vault' and then inside THAT folder is where you're trying to color the folders, that won't work, I just tested it.
> 
> The color needs to be applied to the TOP most folder, only then will it work. Try making a new folder outside of that vault folder or renaming the vault folder itself to one of the colors (I repeat, the vault folder, NOT the vault itself.) Then it'll be whichever color it was selected for it and all subsequent folders inside it. As far as I know you can't color sub folders, only top ones, at least with this snippet.

I think it would be great to have sub folders colored as well, i take notes of things i work on in my current company and i don't want to make a new vault for every company.

my folder structure is as follows:

```
01 - companyname ltd.
    01 - overview
        01 - about
        02 - provided items
        03 - tasks
    02 - selfdocumentation
        2024
           2024-06
              01 - KW 23
              02 - KW 24
              03 - KW 25
              04 - KW 26
```

KW stands for Kalenderwoche (German) aka. Calender Week

**Comment by Rex9988** [comment]
Yeah, if you want different colors for different companies you'd need different folders, not different vaults necessarily, but for coloring subfolders, other than trying to find or create a snippet that does just that, you can only try to ask Cyan for that as a feature.

It'd be a neat thing to have indeed.
EDIT - Although I should mention that if done it should probably be done with different shortcut than the main folder 00, 01 etc, or with a different snippet, for the people that like it as it is or don't want to change the way they organize folders with said shortcuts.

**Comment by christh** [comment]
I had a crack at solving the OP's 3rd September request and made a PR. It works with 01, 02, etc. subfolders. I had a go at making it work with generically named subfolders as suggested by @Rex9988 , but didn't quite succeed. 

As I was working with a 1. 2. 3. type folder structure previously for things I'd decided the order on, to be honest I was happy having the control that only folders starting with 01 to 07 got their own special colours... It was getting a bit rainbow'y in there and kinda distracting!

Hope this helps someone in any case!

**Comment by thorrrr** [comment]
Cheers for help  i dropped messing to time consuming
